### PR TITLE
ARTEMIS-1646 fix for browsing messages sent via JS client

### PR DESCRIPTION
**Pull request by stanlyDoge**

### CODE REVIEW

File: artemis-commons/src/main/java/org/apache/activemq/artemis/utils/collections/TypedProperties.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
 import org.apache.activemq.artemis.utils.AbstractByteBufPool;
 import org.apache.activemq.artemis.utils.ByteUtil;
 import org.apache.activemq.artemis.utils.DataConstants;
+import org.apache.qpid.proton.amqp.UnsignedInteger;
```
Comment (michaelandrepearce) [inline artemis-commons/src/main/java/org/apache/activemq/artemis/utils/collections/TypedProperties.java]: -1 AMQP dependencies should only be within the AMQP Protocol module.

**Comment by michaelandrepearce** [comment]
@stanlyDoge If this needs to be managed / handled for AMQP clients, this must be done within the AMQP protocol module only. We cannot have protocol specifics leaking out their modules.

File: artemis-commons/pom.xml [hunk 2]
```diff
@@ -79,6 +79,10 @@
          <artifactId>junit</artifactId>
          <scope>test</scope>
       </dependency>
+       <dependency>
```
Comment (michaelandrepearce) [inline artemis-commons/pom.xml]: -1 see other comment , AMQP deps must stay within the AMQP module only.

**Comment by stanlyDoge** [comment]
I do not think failed test has any in common with these changes. 

> Failed tests: 
  NetworkHealthTest.testCheckUsingHTTP:213 null

**Comment by michaelandrepearce** [comment]
@stanlyDoge can there be an added test case please for this change

**Comment by michaelandrepearce** [comment]
@stanlyDoge just looking a bit closer at this, also i think you'll need to cover as well, UnsignedLong, UnsignedShort, UnsignedByte.

**Comment by stanlyDoge** [comment]
@michaelandrepearce Good point. Pushing changes & test in few minutes.

**Comment by michaelandrepearce** [comment]
LGTM, i think the PR build issue is resolved, if you could just do a comment change and force push or something, to trigger another build, just to get the green PR before merging.

**Comment by stanlyDoge** [comment]
Done, let's wait for result :) Is there any else way how to kick off build?

**Comment by michaelandrepearce** [comment]
@stanlyDoge thanks, will merge shortly.

**Pull request closed without merging**